lib/ansible/runner/credential/google_credential.rb

```diff
@@ -25,10 +25,13 @@ def write_config_files
 
       def write_gce_credentials_file
         json_data = {
-          :type         => "service_account",
-          :private_key  => auth.auth_key || "",
-          :client_email => auth.userid || "",
-          :project_id   => auth.project || ""
+          :type                        => "service_account",
+          :private_key                 => auth.auth_key || "",
+          :client_email                => auth.userid || "",
+          :project_id                  => auth.project || "",
```

Comment:
@Fryguy do you know where `Authentication#project` is coming from?  I don't see a column `:project` on the `Authentications` table and there are no subclasses of `Authentication` that have a `:project` method as far as I can tell

Reply:
it's in the options as per the specs: https://github.com/ManageIQ/manageiq/blob/master/spec/lib/ansible/runner/credential/google_credential_spec.rb#L22

I'm not sure _how_ that works though.

Reply:
Wow - I missed that too and I had links into that very file 😆